scripts/restore

```diff
@@ -36,8 +36,6 @@ final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 #=================================================
 ynh_script_progression --message="Validating restoration parameters..." --weight=1
 
-ynh_webpath_available --domain=$domain --path_url=$path_url \
-	|| ynh_die --message="Path not available: ${domain}${path_url}"
```

Comment:
Why remove that ? This looks kinda necessary.

Reply:
https://github.com/YunoHost/example_ynh/pull/171

Reply:
ok !